docs/release-notes/next/GH-2845-accounts-state-gc.md

```diff
@@ -0,0 +1,6 @@
+* Adds garbage collector for removing old account states to free up space on disk.
+  By default the GC is disabled, and does the cleanup every 50000 blocks.
```

Comment:
When it is disabled, it should not do the garbage collection at all, not even after 50k blocks

Reply:
Bad wording, will fix this.